### ISSUE TITLE
Publish DWDS custom app

### ISSUE THREAD

**Issue by kelson42** [kiwix/apple#522]
... using the new custom app workflow.

But we have a nice ZIM file of https://www.dwds.de/ and we should created a dedicated custom app on iOS/macOS

Blocked by #520

**Comment by kelson42** [comment]
@BPerlakiH This ticket is the TOP priority. Assigning it to you. Put all the necessary code in a dedicated branch and upload manually to store.

**Comment by kelson42** [comment]
@rgaudin agree if the usage is to have the less important at start of the id. @BPerlakiH What do you think?

**Comment by rgaudin** [comment]
That's not the usage. AFAIK the recommended way is the reversed-tld one so ours should be named `org.kiwix.<something>`.

**Comment by kelson42** [comment]
What about "org.kiwix.custom.dwds"?

**Comment by rgaudin** [comment]
👍

**Comment by BPerlakiH** [comment]
Version 1.0 of the DWDS app has been uploaded to TestFlight. 
I will split my PR into Kiwix only changes to be applied to this repo, and branding changes kept in a separate PR (not to merged into this repo).

**Comment by kelson42** [comment]
Closing this ticket as app has been published (iTunes only so far) and there is no value to keep it open.
